main.js

```diff
@@ -545,3 +554,25 @@ function getCurrency (currencyString, currencySign) {
 	  }
 	});
 }
+function dirtyWords (wordToDefine) {
+	var url = "http://api.urbandictionary.com/v0/define?term="+wordToDefine;
+	request(url, function (err, response, body) {
+		 if(err){
+	    console.log('error getting definition: ', error);
+	
+	  }
+	else {
+    		console.log('Urban Dictionary Definition:', body);
+    		var sample = JSON.parse(body);
+		var definition = sample.list[0]
```

Comment:
This won't work if there aren't any definitions since you're not checking if the definition exists before you assign it to `definition`